lab/README.md

```diff
@@ -16,30 +16,29 @@ The steps below will get you up and running to deploy the Relecloud Django appli
 ## Get Started
 ### Create your own repo from this template
 
-Open the [Azure-Samples/azure-django-postgres-aca](https://github.com/Azure-Samples/azure-django-postgres-aca) repository and click the "Use this Template" button at the top right hand corner.
+Open the [Azure-Samples/azure-django-postgres-aca](https://github.com/Azure-Samples/azure-django-postgres-aca) repository, click the **Use this Template** button at the top right hand corner, and select **Create a new repository**.
```

Comment:
I think learn docs use bold for UI text, so I tried to use that formatting consistently. I dont mind quotes myself so I can revert if you prefer. Bold is nice for skimmers though.